src/lib/index.ts

```diff
@@ -79,17 +91,24 @@ const main = async () => {
 
       if (typeof snykDeltaResults.result !== 'undefined') {
         try {
-          const ghCommitStatusUpdateResponse = await sendCommitStatus(
-            (snykDeltaResults.passIfNoBaseline? 0 : snykDeltaResults.result),
-            snykProjectDetails,
-            githubDetails
-          );
-
           let shouldCommentPr = false;
           if (snykDeltaResults.result > 0 && ghPRNumber) {
             shouldCommentPr = true;
           }
 
+          if(snykDeltaResults.passIfNoBaseline === true 
+            && (snykDeltaResults.newVulns || snykDeltaResults.newLicenseIssues))
+          {
+            shouldCommentPr = true;
+          }
+
+          const ghCommitStatusUpdateResponse = await sendCommitStatus(
+            snykDeltaResults.result,
+            snykProjectDetails,
+            githubDetails, 
+            shouldCommentPr,
```

Comment:
This is not correct. shouldCommentPR can also be set to true regardless of passIfNoBaseline.
Need to add another bollean var to be set to true in in `if(snykDeltaResults.passIfNoBaseline === true  && (snykDeltaResults.newVulns || snykDeltaResults.newLicenseIssues))` so it gets passed to sendCommitStatus only when there is no baseline AND the tests results have vulns.